tests/src/Context/MinkContext.php

```diff
@@ -309,6 +309,31 @@ public function getTableRow(Element $element, $search) {
   }
 
   /**
+   * Compare the position from top between 2 divs by class.
+   *
+   * @param string $div1
+   *   Class of the first div.
+   * @param string $div2
+   *   Class of the second div.
+   *
+   * @throws \Exception
+   *    Throw exception if the two positions from top are different.
+   *
+   * @Then I check if :div1 and :div2 have the same position from top
+   */
+  public function checkIfTwoDivHaveSamePosition($div1, $div2) {
+    $javascript1 = "return jQuery('." . $div1 . "').offset().top;";
+    $result_div1 = intval($this->getSession()->evaluateScript($javascript1));
+
+    $javascript2 = "return jQuery('." . $div2 . "').offset().top;";
+    $result_div2 = intval($this->getSession()->evaluateScript($javascript2));
+
+    if ($result_div1 != $result_div2) {
```

Comment:
Here I would prefer to use 'aasert' function as follow:
`assert($result_div1, equals($result_div2));`
It will simplify the step implementation to make it more readable and easier to maintain in future. This will also make the step more consistent with the recently developed steps implementations.